openslides/urls.py

```diff
@@ -11,14 +11,15 @@
 urlpatterns = patterns(
     '',
     url(r'^rest/', include(router.urls)),
-    # TODO: add "special" urls, for example pdf views etc.
+    (r'^users/', include('openslides.users.urls')),
```

Comment:
I don't see a benefit for it. For normal urls we want to use the url_pattern and that has to be done in the url-form. But we do not need it for include urls.

Also, this change is not a part of this pull request. Please open a issue in which we can talk about it.